src/background.js

```diff
@@ -228,12 +228,10 @@ const rollout = {
     switch (policyEnableDoH) {
     case "enable_doh":
       log("Policy requires DoH enabled.");
-      await stateManager.setState("enabled");
       browser.experiments.heuristics.sendHeuristicsPing(policyEnableDoH, results);
       break;
     case "disable_doh":
       log("Policy requires DoH to be disabled.");
-      await stateManager.setState("disabled");
```

Comment:
This is problematic. `checkEnterprisePolicies()` can return `"disable_doh"` when the policy doesn't mention DoH at all. We do want to disable DoH in that case, removing that line affects that logic.

Reply:
If their policy is disabled, it would set the trr.mode to 5, but the setState will override it to 0. Do we need to add more logic to the API for policy detection?

Reply:
This could be solved by adding the two lines back in, but put a conditional around each line: we only set enable/disable if user hasn't set trr.mode.... A few lines above, we already check for that, so we can only get here is user hasn't set trr.mode -> it's safe to add the lines back in.